### PR TITLE
+ Added Livestatus Listener object. #7222

### PR DESCRIPTION
**Pull request by cropalato**

### CODE REVIEW

**Comment by nickchappell** [comment]
Just for reference,  [#7222](https://dev.icinga.org/issues/7222) is the [dev.icinga.org](https://dev.icinga.org) issue.

**Comment by nickchappell** [comment]
Hi @cropalato, the patch for this won't apply because the README has diverged from the state it was in when you made the PR (I've merged a few other PRs tonight). Would you be able to rebase the current master onto your feature branch and re-push?

This page is a good overview on how to do that: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#how-do-i-rebase

File: manifests/object/livestatuslistener.pp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+define icinga2::object::livestatuslistener (
+  $ensure                         = 'file',
+  $object_livestatuslisternername = $name,
+  $socket_type                    = undef,
+  $bind_host                      = undef,
+  $bind_port                      = undef,
+  $socket_path                    = undef,
+  $compat_log_path                = undef,
+  $target_dir                     = '/etc/icinga2/objects/livestatuslistener',
```
Comment (nickchappell) [inline manifests/object/livestatuslistener.pp]: This should be `/etc/icinga2/objects/livestatuslisteners`, with an `s` at the end of `livestatuslistener`

**Comment by nickchappell** [comment]
Hi @cropalato,

Found a small issue, left an inline comment on the diff.

**Comment by cropalato** [comment]
Fixed typo

**Comment by nickchappell** [comment]
Merged this in. Thanks @cropalato!

**Pull request closed without merging**